### DISCUSSION TITLE
How to use RSA implicit rejection padding API ossl_rsa_padding_check_PKCS1_type_2() outside the OpenSSL library ?

### DISCUSSION THREAD

**Discussion by venkatesh6911** [openssl/openssl#24363] [Q&A] [answered]
I understand OpenSSL enables implicit rejection by default. So, I am wondering how can I invoke the padding implementation from my test application. It is not exposed outside the RSA crypto library.

**Comment by paulidale** [comment, marked as answer]
That's an internal function with zero API guarantees.  You do not want to use this in any kind of production environment (& I'd include testing as part of one).

The safest option would be to copy the relevant code to your application.
Failing that, you could link against a static build of OpenSSL where these symbols are visible.
